config/environment.rb

```diff
@@ -34,7 +34,7 @@
     migrations 'db/migrations'
     schema     'db/schema.sql'
 
-    logger "log/development.log", level: :debug
+    logger "log/#{Hanami.env}.log", level: :debug
```

Comment:
It may be useful to check queries in development without noise after tests. If Heroku has issues about it, u can just close this pr. It is not necessary to change ;)